libs/user-facing-errors/src/quaint.rs

```diff
@@ -179,7 +179,7 @@ pub fn render_quaint_error(kind: &ErrorKind, connection_info: &ConnectionInfo) -
                 Some(KnownError::new(common::DatabaseOperationTimeout {
                     time: "N/A".into(),
                     context: format!(
-                        "Socket timeout (the database failed to respond to a query within the configured timeout — see https://pris.ly/d/sqlite-connector for more details.). Database: {}",
+                        "The database failed to respond to a query within the configured timeout — see https://pris.ly/d/sqlite-connector for more details. Database: {}",
```

Comment:
What can I see at that URL that would help in this case?

Reply:
Ah, I see this is just an adapted error message from all the others for PostgreSQL, MySQL and SQL Server where we probably actually document parameters to change the timeout?

Reply:
Yes, the message is adapted from the other providers' messages.

Reply:
So does this message any sense when the page does not actually include any more details? What details _could_ there be?